basic/src/test/java/org/jboss/hal/testsuite/test/configuration/deploymentScanners/DeploymentScannerTestCase.java

```diff
@@ -0,0 +1,225 @@
+package org.jboss.hal.testsuite.test.configuration.deploymentScanners;
+
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.dmr.ModelNode;
+import org.jboss.hal.testsuite.category.Standalone;
+import org.jboss.hal.testsuite.cli.CliClient;
+import org.jboss.hal.testsuite.cli.CliClientFactory;
+import org.jboss.hal.testsuite.cli.TimeoutException;
+import org.jboss.hal.testsuite.dmr.Dispatcher;
+import org.jboss.hal.testsuite.dmr.ResourceAddress;
+import org.jboss.hal.testsuite.dmr.ResourceVerifier;
+import org.jboss.hal.testsuite.fragment.shared.modal.ConfirmationWindow;
+import org.jboss.hal.testsuite.page.config.DeploymentScannerPage;
+import org.jboss.hal.testsuite.util.Console;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import static org.junit.Assert.assertFalse;
+
+/**
+ * Created by pcyprian on 22.10.15.
+ */
+@RunWith(Arquillian.class)
+@Category(Standalone.class)
+public class DeploymentScannerTestCase {
+    private static final String NAME = "test";
+    private static final String PATH = "../standalone/deployments";
+
+    private ModelNode path = new ModelNode("/subsystem=deployment-scanner/scanner=" + NAME);
+    private ResourceAddress address = new ResourceAddress(path);
+    Dispatcher dispatcher = new Dispatcher();
```

Comment:
`dispatcher` should be closed after the tests. You can achieve this e.g. by making `dispatcher` static and close it within `@AfterClass` method.